keras_hub/src/models/retinanet/prediction_head.py

```diff
@@ -0,0 +1,148 @@
+import keras
+
+
+class PredictionHead(keras.layers.Layer):
+    """The classification/box predictions head.
+
+    Arguments:
+        output_filters: int. Number of convolution filters in the final layer.
```

Comment:
you have mentioned that ` either the classification or the box regression head is returned based on output_filters`. Can you specify what the `output_filters` value  needs to be to expect the two different type of output.

Reply:
Done